chart/values.schema.json

```diff
@@ -253,6 +253,11 @@
                     "description": "Specifies whether RBAC resources should be created.",
                     "type": "boolean",
                     "default": true
+                },
+                "createSCCRoleBinding": {
+                    "description": "Specifies whether SCC RoleBinding resource should be created.",
```

Comment:
I think it would be great to add a little more explanation of what SCC is. Maybe a small chapter explaining in Production Guide (https://airflow.apache.org/docs/helm-chart/stable/production-guide.html) and link to it from here (and refer to the parameter from there)?  Also a comment that it is related to OpenShift wiould be great.

Reply:
Fair enough, I'm too good at documentation but I will do my best.

Reply:
Documentation in place!